src/sentry/migrations/0205_semver_backfill.py

```diff
@@ -31,7 +31,7 @@ def convert_build_code_to_build_number(build_code):
     revision = data.revision,
     prerelease = data.prerelease,
     build_code = data.build_code,
-    build_number = data.build_number
+    build_number = data.build_number::bigint
```

Comment:
I tested this with a null `build_number` and it works as expected - write a null value to the column